eks/ibl_pupil_smoother.py

```diff
@@ -305,133 +308,175 @@ def ensemble_kalman_smoother_ibl_pupil(
     return markers_df, s_finals
 
 
+@typechecked
 def pupil_optimize_smooth(
```

Comment:
as above, rename to `pupil_run_kalman_smoother` (or whatever, as long as it parallels the new name in core.py)